app/views/users/_edit.haml

```diff
@@ -5,242 +5,241 @@
 - content_for :page_title do
   = t('.edit_account')
 
+= render 'shared/settings_nav'
+
 .container-fluid
-  = render 'shared/settings_nav'
+  .col-lg-8.col-lg-offset-2
+    .row
+      .col-md-6
+        %h3= t('.your_handle')
 
-  .container-fluid
```

Comment:
This adds some inconsistency for the desktop views. /user/edit has no surrounding `.container-fluid` anymore while all other settings pages still have one.